src/app/views/river-index/components/nwi-map.vue

```diff
@@ -460,6 +487,21 @@ export default {
       })
     },
     initMap () {
+      if (isAmplifyDeploy && this.tileserverToken === null) {
+        const url = 'https://tileserver.americanwhitewater.org/token/';
+        const that = this;
```

Comment:
I am perhaps exposing myself as a former "server-side" developer here and not a Real Javascript Developer with my uncertainty but I *believe* you can skip this step. Using arrow functions (as you do below) binds `this` through each callback. Try it and see, it's possible I'm wrong!

Reply:
I agree that it would be better to not do this. I remember we worked hard in React to remove the need for this but it wasn't working here so I just gave up :P